app/scripts/views/mixins/signin-mixin.js

```diff
@@ -96,6 +96,41 @@ define(function (require, exports, module) {
 
       return this.invokeBrokerMethod(brokerMethod, account)
         .then(this.navigate.bind(this, this.model.get('redirectTo') || 'settings', {}, navigateData));
+    },
+
+    /*anchor tag present in both signin and signup views*/
+    events: {
+      'click #suggest-sync a': 'onSuggestSyncClick'
+    },
+
+    /**
+     * Sets showSyncSuggestion var in view context (signin and signup)
+     * sends log information about suggestion panel visibility
+     * @method setShowSyncSuggestion
+     * @param {object} context: view context object
+    */
+
+    setShowSyncSuggestion: function (context) {
+      if (! this.relier.get('service') && ! context.isSyncMigration && ! context.isAmoMigration) {
+        context.showSyncSuggestion = true;
+        this.metrics.logEvent('sync-suggest.visible');
```

Comment:
hey, no need to duplicate. if you call `this.metrics.logViewEvent('sync-suggest.visible');`
it will log `fxa.content.screen.signup.sync-suggest.visible` or `fxa.content.screen.signin.sync-suggest.visible` depending on the view you are on. This will let us see exactly which screen gets more usage 👍